joomla/joomla-cms
# [com_actionlogs] Fix plugin links

Pull Request for Issue #23841 .

### Summary of Changes

Pass `$urlVar` parameter to `getContentTypeLink()`.

### Testing Instructions

Unpublish/publish a plugin
Return to control panel and see the message in the Action Log module
Follow the link

### Expected result

com_plugins opens with the specified plugin open for editing

### Actual result

com_plugin opens at the list

### Documentation Changes Required

No.
